homeassistant/components/sensor/lirc.py

```diff
@@ -0,0 +1,108 @@
+"""
+LIRC interface to receive signals from a infrared remote control.
+
+This sensor will momentarily set state to various values as defined
+in the .lintrc file which can be interpreted in home-assistant to
+trigger various actions.
+
+Sending signals to other IR receivers can be accomplished with the
+shell_command component and the irsend command.
+"""
+
+import threading
+import time
+import logging
+
+from homeassistant.helpers.entity import Entity
+from homeassistant.const import EVENT_HOMEASSISTANT_STOP
+
+LIRC = None
+
+_LOGGER = logging.getLogger(__name__)
+ICON = 'mdi:remote'
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup LIRC capability."""
+    # Perform safe import of third-party python-lirc module
+    try:
+        import lirc
+        global LIRC
```

Comment:
Yes, I can just import it down below where I need it again. Uh oh, bad name choice on my part. The site-package module `lirc` has the same name as this module.